msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -807,6 +809,28 @@ public void acquireTokenSilentAsync(@NonNull final String[] scopes,
         MSALApiDispatcher.submitSilent(silentTokenCommand);
     }
 
+    private String getSilentRequestAuthority(final IAccount account) {
```

Comment:
Why wouldn't we move this method to the authority class?

Reply:
Which public API are you referring to?  I meant why wouldn't we move getSilentRequestAuthority to the Authority class?  The Authority class is in the internal package... hence not part of the public API

Reply:
Ah, I got what you mean here. I will make the fix after release.